internal/commands/app/create_inputs_test.go

```diff
@@ -209,14 +215,138 @@ func TestAppCreateInputsResolveDirectory(t *testing.T) {
 		assert.Equal(t, "new-app", inputs.LocalPath)
 	})
 
-	t.Run("should error when path specified is another realm app", func(t *testing.T) {
+	t.Run("should fail when realm app already exists in current wd", func(t *testing.T) {
 		profile, teardown := mock.NewProfileFromTmpDir(t, "app_create_test")
 		defer teardown()
 
-		specifiedDir := "test-dir"
-		inputs := createInputs{LocalPath: specifiedDir}
+		_, ui := mock.NewUI()
+
+		existingApp := realm.App{
+			ID:          primitive.NewObjectID().Hex(),
+			GroupID:     primitive.NewObjectID().Hex(),
+			ClientAppID: "existing-app-abcde",
+			Name:        "existing-app",
+		}
+
+		rc := mock.RealmClient{}
+		rc.CreateAppFn = func(groupID, name string, meta realm.AppMeta) (realm.App, error) {
+			return existingApp, nil
+		}
+		rc.ImportFn = func(groupID, appID string, appData interface{}) error {
+			return nil
+		}
+
+		inputs := createInputs{newAppInputs: newAppInputs{
+			Name:    existingApp.Name,
+			Project: existingApp.GroupID,
+		}}
+		cmd := &CommandCreate{inputs}
+		assert.Nil(t, cmd.Handler(profile, ui, cli.Clients{Realm: rc}))
+
+		existingDir := filepath.Join(profile.WorkingDirectory, existingApp.Name)
+		dir, err := inputs.resolveLocalPath(ui, existingDir)
+
+		assert.Equal(t, errProjectExists{existingDir}, err)
+		assert.Equal(t, dir, "")
+	})
+
+	t.Run("should fail when realm app already exists in newly specified path", func(t *testing.T) {
+		profile, teardown := mock.NewProfileFromTmpDir(t, "app_create_test")
+		defer teardown()
+
+		_, console, _, ui, err := mock.NewVT10XConsole()
+		assert.Nil(t, err)
+		defer console.Close()
+
+		existingApp := realm.App{
+			ID:          primitive.NewObjectID().Hex(),
+			GroupID:     primitive.NewObjectID().Hex(),
+			ClientAppID: "existing-app-abcde",
+			Name:        "existing-app",
+		}
+
+		rc := mock.RealmClient{}
+		rc.CreateAppFn = func(groupID, name string, meta realm.AppMeta) (realm.App, error) {
+			return existingApp, nil
+		}
+		rc.ImportFn = func(groupID, appID string, appData interface{}) error {
+			return nil
+		}
+
+		inputs := createInputs{newAppInputs: newAppInputs{
+			Name:    existingApp.Name,
+			Project: existingApp.GroupID,
+		}}
+		cmd := &CommandCreate{inputs}
+		assert.Nil(t, cmd.Handler(profile, ui, cli.Clients{Realm: rc}))
+
+		doneCh := make(chan (struct{}))
+		go func() {
+			defer close(doneCh)
+			console.ExpectString("Local path './existing-app' already exists, writing app contents to that destination may result in file conflicts.")
+			console.ExpectString("Would you still like to write app contents to './existing-app'? ('No' will prompt you to provide another destination)")
+			console.SendLine("no")
+			console.ExpectString("Local Path")
+			console.SendLine("existing-app")
+			console.ExpectEOF()
+		}()
+
+		dir, err := inputs.resolveLocalPath(ui, profile.WorkingDirectory)
```

Comment:
not a big deal and no need to make changes, but generally (here and the test above) if you're testing the `err` case, we usually don't care about the other return values (e.g. `_, err := inputs.resolveLocalPath(...)`